css/selectors.json

```diff
@@ -199,6 +199,30 @@
     ],
     "status": "experimental"
   },
+  ":host": {
+    "syntax": ":host",
+    "groups": [
+      "Pseudo-classes",
+      "Selectors"
+    ],
+    "status": "standard"
+  },
+  ":host()": {
+    "syntax": ":host( <compound-selector-list> )",
```

Comment:
compound-selector-list should be defined in syntaxes.json

Reply:
I did that, but inside this PR: https://github.com/mdn/data/pull/169

So I guess we need to figure out that one first?

Reply:
@teoli2003 did you have any more follow up on this?